src/ContextFreeTasks.Shared/ContextFreeTask.cs

```diff
@@ -11,5 +11,6 @@ public struct ContextFreeTask
         public Task Task => _task ?? FromResult(default(object));
         internal ContextFreeTask(Task t) => _task = t;
         public ContextFreeTaskAwaiter GetAwaiter() => new ContextFreeTaskAwaiter(Task);
+        public void Wait() => _task?.Wait();
```

Comment:
This is by design. Users should use `GetAwaiter().GetResult()` instead of `Wait()`.